src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -108,4 +129,104 @@ private void addListingToBusinesses(PlacesSearchResult place) {
     allBusinesses.add(new Listing(name, formattedAddress, 
           placeLocation, rating, photos, types));
   }
-}
\ No newline at end of file
+
+  public void checkNumberOfLocationsOfBusiness() {
+    GeoApiContext context = 
+      new GeoApiContext.Builder().apiKey(KEY).build();
+    numberOfSmallBusinesses = 0;
+    Iterator<Listing> businesses =  allBusinesses.iterator();
+    while(businesses.hasNext()
+          && numberOfSmallBusinesses < SMALL_BUSINESSES_DISPLAYED) {
+      Listing currentBusiness = businesses.next();
+      TextSearchRequest request = 
+        new TextSearchRequest(context).query(currentBusiness.getName())
+          .location(latLng).radius(50000);
+      try {                                       
+        PlacesSearchResult[] similarBusinessesInTheArea = 
+          request.await().results;
+        if (similarBusinessesInTheArea.length > 1){
+          checkBusinessThroughLinkedin(currentBusiness, similarBusinessesInTheArea);
+        }else if(similarBusinessesInTheArea.length == 1){
+          numberOfSmallBusinesses++;
+        }
+      } catch(GeneralSecurityException | IOException | InterruptedException | ApiException e ) {
+          LOGGER.warning(e.getMessage());
+        } 
+    }
+  }
+  
+  private void checkBusinessThroughLinkedin(Listing currentBusiness, 
+                          PlacesSearchResult[] similarBusinessesInTheArea) 
+                              throws GeneralSecurityException, IOException {
+    String cx = dotenv.get("CX"); 
+    Customsearch cs = new Customsearch.Builder(
+        GoogleNetHttpTransport.newTrustedTransport(), 
+        JacksonFactory.getDefaultInstance(), null) 
+              .setApplicationName("linkedinSearch") 
+              .setGoogleClientRequestInitializer(new CustomsearchRequestInitializer(KEY)) 
+              .build();
+
+    Customsearch.Cse.List list = cs.cse().list(currentBusiness.getName()).setCx(cx); 
+    List<Result> searchJsonResults = list.execute().getItems();
+    String[] numberOfFollowers;
+    int companyFollowers = 0;
+    if (searchJsonResults != null && searchJsonResults.size() != 0) {
+      Result linkedinBusiness = searchJsonResults.get(0);
+      String businessDescription = 
+        (String) linkedinBusiness.getPagemap().get("metatags").get(0).get("og:description");
+      if(businessDescription.contains(START_SUBSTRING) == true 
```

Comment:
nit:  Add a space above this line to make this more readable.

Also, you can remove the "== true" part if you want. The evaluation will still work. Apologies for missing that in my last round.